### PR TITLE
#31 app store app component

### LINKED ISSUE #31: Stworzenie komponentu widoku aplikacji
kafelek + jego szczegóły

### PR DESCRIPTION
**Pull request by maciekkoz98**
App component to show in the AppStore

### CODE REVIEW

**Review by KonradMagiera — APPROVED** [review]
In `AppCard` interface field `description` could be optional but other than that, it's fine

**Review by JCeluch — APPROVED** [review]

File: frontend/compute-system/package.json [hunk 1]
```diff
@@ -5,7 +5,6 @@
   "dependencies": {
     "@material-ui/core": "^4.11.0",
     "@material-ui/icons": "^4.9.1",
-    "@material-ui/styles": "^4.10.0",
```
Comment (lukitoki1) [inline frontend/compute-system/package.json:8]: why?

File: frontend/compute-system/src/components/AppStore/components/AppCard.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                <Typography variant="caption">Rate</Typography>
+              </Grid>
+            </Grid>
+          </Grid>
+          <Grid item>
+            <Button className={classes.addButton}>Add to cockpit</Button>
+          </Grid>
+        </Grid>
+      </Grid>
+    </Paper>
```
Comment (lukitoki1) [inline frontend/compute-system/src/components/AppStore/components/AppCard.tsx]: a little messy... could think of breaking this into multiple small components

File: frontend/compute-system/src/components/AppStore/components/appCardStyles.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      width: "750px",
+    },
+    updatedText: {
+      marginBottom: theme.spacing(1),
+    },
+    usedRateBox: {
+      marginBottom: theme.spacing(2),
+    },
+  })
+);
```
Comment (lukitoki1) [inline frontend/compute-system/src/components/AppStore/components/appCardStyles.ts]: same as with the AppCard component

File: frontend/compute-system/src/mocks/AppStore/AppCard/mockAppCards.ts [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import AppCard from "../../../components/AppStore/interfaces/appCard";
```
Comment (lukitoki1) [inline frontend/compute-system/src/mocks/AppStore/AppCard/mockAppCards.ts]: also works without "../../../"

**Review by lukitoki1 — COMMENTED** [review]
Overall it could be divided into several separate commits:

- you added theme
- you cleaned some mess from initial React project,
- you initialized routing,
- and finally: you added component with mock data.

But these are really just details, so do as you wish, for me it's merge'able.

File: frontend/compute-system/package.json (hunk 1 above)
Comment (maciekkoz98) [reply frontend/compute-system/package.json:8]: well i guess not needed ;p

File: frontend/compute-system/src/components/AppStore/components/AppCard.tsx (hunk 2 above)
Comment (maciekkoz98) [reply frontend/compute-system/src/components/AppStore/components/AppCard.tsx]: true

File: frontend/compute-system/src/mocks/AppStore/AppCard/mockAppCards.ts (hunk 4 above)
Comment (maciekkoz98) [reply frontend/compute-system/src/mocks/AppStore/AppCard/mockAppCards.ts]: nope, that's different catalog

**Pull request merged**